Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml

```diff
@@ -36,7 +37,11 @@
     - WhitelistChameleon
 
 - type: entity
-  parent: [ClothingShoesBootsMag, BaseGrandTheftContraband]
+  parent: [ClothingShoesBootsMagBase, BaseEngineeringContraband]
```

Comment:
do magboots even qualify as engineering contraband to begin with? They're available in quite a few other places like salv and general EVA?

Reply:
If magboots are not supposed to be engineering contraband, that is something that can be changed easily. Currently, every type of magboots is listed as engineering contraband. The PR was to fix the egregious issues (namely, bloodreds not being syndicate contraband).